led_strip/examples/led_strip_spi_ws2812/main/idf_component.yml

```diff
@@ -3,4 +3,3 @@ dependencies:
   espressif/led_strip:
     version: '^2.4'
     override_path: '../../../'
-  idf: ">=5.1"
```

Comment:
this is still needed becase now led_strip acquires only esp-idf 5.0, but this example can only build for 5.1